rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -1848,12 +1848,29 @@ void main_window::SaveWindowState() const
 	// Save gui settings
 	m_gui_settings->SetValue(gui::mw_geometry, saveGeometry(), false);
 	m_gui_settings->SetValue(gui::mw_windowState, saveState(), false);
-	m_gui_settings->SetValue(gui::mw_mwState, m_mw->saveState(), true);
 
-	// Save column settings
-	m_game_list_frame->SaveSettings();
-	// Save splitter state
-	m_debugger_frame->SaveSettings();
+	// NOTE:
```

Comment:
This entire comment is not necessary.
There's no reason to justify checking the validity of the pointers.

Reply:
that comment is more to highlight the use cases that can be covered by that method